src/tangram/app.py

```diff
@@ -43,7 +44,8 @@ async def shutdown_publish_job() -> None:
     log.info("shutdown - publish job done")
 
 
-templates = Jinja2Templates(directory="templates")
+tangram_module_root = pathlib.Path(__file__).resolve().parent
+templates = Jinja2Templates(directory=tangram_module_root / "templates")
```

Comment:
can we do something similar for the log.yml file?

Reply:
@xoolive Sorry, missed this.

I'm thinking of adding a config file, `.toml` for example, so plugins can have their sections too. And by default it's placed in the package root directory.